docs/data/routes/docs/nextjs/introduction/why-nextjs/en.md

```diff
@@ -3,4 +3,33 @@ name: why-nextjs
 routeTemplate: ./data/component-templates/article.yml
 title: Why Next.js and JSS
 ---
-# Why Next.js and JSS
\ No newline at end of file
+# Why Next.js and JSS?
+
+[Next.js](https://nextjs.org/) is a framework built on top of **React**. It aims to help developers create production-ready applications with minimal need for configuration and boilerplate code. 
+
+The latest JSS SDK leverages many of Next.js' [features](https://nextjs.org/#features), to simplify and enhance development workflow and experience. 
+
+## Improvements over previous JSS SDKs
+
+* Development and production runtime parity, including SSR during development.
+* Fewer confusing “modes” and deployment requirements.
+* No Headless SSR Proxy needed for production or for integration with Sitecore tracking.
+* No need to deploy your application bundle to the Content Management instance for Experience Editor support.
+
+## Features
+
+- Support for Next.js static HTML export, incremental static site generation, server-side rendering, and hybrid rendering.
+- Dynamic Sitecore-based routing via Next.js pages and the Sitecore Layout Service.
```

Comment:
Sitecore-based routing = author-defined URL structure. Should we say that here or leave it for the routing section?

Reply:
Changed, that's clearer thanks!